pom.xml

```diff
@@ -48,6 +50,11 @@
     </dependencies>
   </dependencyManagement>
   <dependencies>
+    <dependency>
+      <groupId>com.github.edwgiz</groupId>
+      <artifactId>maven-shade-plugin.log4j2-cachefile-transformer</artifactId>
+      <version>2.1</version>
+    </dependency>
```

Comment:
This should not be specified as a dependency of this project, as it is only a dependency of the shade plugin, in order to add the transformer to that.